src/styles/_cta.scss

```diff
@@ -151,6 +151,26 @@ a.cta {
     opacity: 1;
 }
 
+.transparent-white {
+    background: transparent 0% 0% no-repeat padding-box;
+    box-shadow: 0px 13px 26px #00956341;
+    border: 1px solid #FFFFFF;
+    border-radius: 0%;
+    letter-spacing: 0;
+    color: #FFFFFF;
```

Comment:
You can just use #fff instead of #fffff. So it will be lessen in the code.

Reply:
changed to #fff